src/memphis.ts

```diff
@@ -56,6 +56,8 @@ const storageTypes: IStorageTypes = {
 
 const maxBatchSize = 5000
 
+let connectionOpts
```

Comment:
why is it global?

Reply:
because in the catch clause this var dont exisst so i declare it to be able to use it in the catch clause